packages/cli/src/models/config/ZosConfig.ts

```diff
@@ -17,16 +17,25 @@ interface Config {
   compilers?: CompilersInfo;
 }
 
-interface Network {
+interface NetworkCamel<T> {
+  networkId: T;
+}
+
+interface NetworkSnake<T> {
+  network_id: T;
+}
+
+type NetworkId<T> = NetworkCamel<T> | NetworkSnake<T> | (NetworkCamel<T> & NetworkSnake<T>);
```

Comment:
Why is the `(NetworkCamel<T> & NetworkSnake<T>)` needed? Isn't that intersection empty?

Reply:
Also, I'd just embed optional network_id and networkId directly in the Network type instead of defining separate interfaces.